### PR TITLE
Update DG appendix

### LINKED ISSUE #196: Update appendix for delete and remind
Add manual testing steps for delete and remind.

Fix bug in manual testing for launching app to take into account of auto remind.

### PR DESCRIPTION
**Pull request by TY1Fan**
Closes #196 

Changes:

* Add manual testing steps for remind and delete.
* Updated the manual testing for launch to include a reminder when app is launched.

### CODE REVIEW

**Review by teamotitan — COMMENTED** [review]
Under #195, I think CF made a good point about using 1. instead of 1. 2. 3. 
ty 🥇

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -467,9 +467,13 @@ testers are expected to do more *exploratory* testing.
 
 1. Initial launch
 
-   1. Download the jar file and copy into an empty folder
+   1. Download the jar file and copy into an empty folder.
+   
+   2. Open a command terminal, cd into the folder that you put the jar file in.
```
Comment (Feng1231) [inline docs/DeveloperGuide.md:472]: All ordered list should be `1.`

Comment (Feng1231) [inline docs/DeveloperGuide.md:472]: I think the cd can be `cd`

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -467,9 +467,13 @@ testers are expected to do more *exploratory* testing.
...
 
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
+   3. Run the jar file with the command in the terminal java -jar ugteach.jar
```
Comment (Feng1231) [inline docs/DeveloperGuide.md:474]: the command can be wrapped in the code too

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -478,24 +482,49 @@ testers are expected to do more *exploratory* testing.
...
 
-   1. Test case: `delete 0`<br>
-      Expected: No student is deleted. Error details shown in the status message. Status bar remains the same.
+   3. Test case: `delete 0`<br>
+      Expected: No student is deleted. UGTeach displays error message.
 
-   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
-      Expected: Similar to previous.
+   4. Other incorrect delete commands to try: `delete`, `delete x` (where x is larger than the list size)<br>
+      Expected: No student is deleted. UGTeach displays error message.
```
Comment (Feng1231) [inline docs/DeveloperGuide.md:498]: Hmm, may I ask if we are gonna standardize on `Similar to previous.` or just repeat the Expected from previous message? I would have to update my PR accordingly too

**Review by Feng1231 — CHANGES_REQUESTED** [review]
The previous ordered list is the suggested convention.
Refer to [here](https://se-education.org/guides/conventions/markdown.html)

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (TY1Fan) [reply docs/DeveloperGuide.md]: Thanks for pointing this out!

Comment (TY1Fan) [reply docs/DeveloperGuide.md]: Thanks for the suggestion!

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (TY1Fan) [reply docs/DeveloperGuide.md]: Thanks for the suggestion! Ive wrapped both of it.

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (TY1Fan) [reply docs/DeveloperGuide.md]: I think we can standardise to similar to previous if doing so does not cause ambiguity

Comment (Feng1231) [reply docs/DeveloperGuide.md]: I see

**Review by Feng1231 — APPROVED** [review]
LGTM

**Pull request merged**